src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -279,12 +279,12 @@ public async Task InitializeAsync(CancellationToken cancellationToken = default)
                 IEnumerable<FunctionMetadata> functionMetadataList = GetFunctionsMetadata();
 
                 string workerRuntime = _environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionWorkerRuntime);
-                _logger.FunctionsWorkerRuntimeValue(workerRuntime);
+                _logger.FunctionsWorkerRuntimeValue(Sanitizer.Sanitize(workerRuntime));
```

Comment:
Just curious - what kind of sensitive information the worker runtime could contain?